src/components/BillCard/SmallBillCard/SmallBillCard.jsx

```diff
@@ -1,16 +1,39 @@
 import React from "react";
+import PropType from "prop-types";
+import {FaBus, FaCar, FaQuestion, FaShoppingBag, FaShoppingCart, FaUtensils } from "react-icons/fa";
+
+import { filterDateTime } from '../../../helpers/DateTime';
+
 import "./styles.scss";
 
-const SmallBillCard = ({ activeBill, bill, filterDateTime, billIcons }) => (
+const getBillIcons = (category) => {
+  const color = "rgba(0, 0, 0, 0.96)";
+  switch (category) {
+    case "food":
+      return <FaUtensils color={color} size={24} />;
+    case "transport":
+      return <FaCar color={color} size={24} />;
+    case "public-transport":
+      return <FaBus color={color} size={24} />;
+    case "grocery":
+      return <FaShoppingCart color={color} size={24} />;
+    case "shopping":
+      return <FaShoppingBag color={color} size={24} />;
+    default:
+      return <FaQuestion color={color} size={24} />;
+  }
+};
+
+const SmallBillCard = ({ activeBillId, bill }) => (
   <div
     className="bill__items card-body"
     id={
-      activeBill && activeBill.id === bill.id
+      typeof activeBillId === 'number' && activeBillId === bill.id
```

Comment:
does it have to be id

Reply:
Ideally yes. How else would you uniquely identify bills?

Reply:
Ah it's a prop rather than a css thing that's why I said it. Less CSS id